docs/reference/platforms/unix-prerequisites.rst

```diff
@@ -5,31 +5,37 @@ some of the common alternatives:
     The package list should be the same as in ci.yml, and the BeeWare tutorial
     (CI will also have WebView requirements)
 
-**Ubuntu 18.04+ / Debian 11+**
+**Ubuntu / Debian**
 
 .. code-block:: console
 
     (venv) $ sudo apt update
-    (venv) $ sudo apt install pkg-config python3-dev libgirepository1.0-dev libcairo2-dev libcanberra-gtk3-module
+    (venv) $ sudo apt install gcc git pkg-config python3-dev gir1.2-gtk-3.0 libgirepository1.0-dev libcairo2-dev libcanberra-gtk3-module
 
 **Fedora**
 
 .. code-block:: console
 
-    (venv) $ sudo dnf install pkg-config python3-devel gobject-introspection-devel cairo-gobject-devel libcanberra-gtk3
+    (venv) $ sudo dnf install gcc git pkg-config python3-devel gtk3 gobject-introspection-devel cairo-gobject-devel libcanberra-gtk3
 
 **Arch / Manjaro**
 
 .. code-block:: console
 
-    (venv) $ sudo pacman -Syu git pkgconf gobject-introspection cairo libcanberra
+    (venv) $ sudo pacman -Syu gcc git pkgconf python3 gtk3 gobject-introspection cairo libcanberra
+
+**OpenSUSE Tumbleweed**
+
+.. code-block:: console
+
+    (venv) $ sudo zypper install gcc git pkgconf-pkg-config python3-devel gtk3 'typelib(Gtk)=3.0' gobject-introspection-devel cairo-devel libcanberra-gtk3-0
```

Comment:
Dockerfile version uses a second "pattern" install for `devel_basis`, rather than an explicit `gcc`.